module/module_test.go

```diff
@@ -188,7 +188,7 @@ func TestModuleFunctions(t *testing.T) {
 
 	m.SetReady(false)
 
-	resp, err := client.Ready(ctx, &pb.ReadyRequest{})
+	resp, err := client.Ready(ctx, &pb.ReadyRequest{ParentAddress: parentAddr})
```

Comment:
We can no longer send `ReadyRequest`s without a `ParentAddress` to modules in tests.